src/java/boa/compiler/SymbolTable.java

```diff
@@ -192,6 +192,7 @@
 
 		// set functions
 		globalFunctions.addFunction("contains", new BoaFunction(new BoaBool(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaTypeVar("V") }, "${0}.contains(${1})"));
+		globalFunctions.addFunction("containsAll", new BoaFunction(new BoaBool(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaSet(new BoaTypeVar("V"))}, "${0}.containsAll(${1})"));
```

Comment:
This is a good point.  But perhaps a better point is: what **is** our naming convention?  The majority seem to follow no uppercase, no underscore.  But I see one with an underscore.  So should this be 'containsall'?  Also, similar issue with the other new function 'getValue'.